src/Components/Aspire_Components_Progress.md

```diff
@@ -22,6 +22,7 @@ As part of the .NET Aspire November preview, we want to include a set of .NET As
 | StackExchange.Redis.OutputCaching       |                  ✅                  |             ✅             |                           N/A                            |              ✅              |          ✅          |        ✅            |         ❌           |              ✅                  |
 | RabbitMQ                                |                  ✅                  |             ✅             |                           ✅                            |              ✅              |                      |                      |         ❌           |              ✅                  |
 | MySqlConnector                          |                  ✅                  |                           |                           ✅                            |              ✅              |          ✅          |        ✅            |         ✅           |              ✅                  |
+| Oracle.EntityFrameworkCore.Database     |                  ✅                  |                           |                           ✅                            |              ✅              |          ✅          |        ✅            |         ✅           |              ✅                  |
```

Comment:
Can you also update https://github.com/dotnet/aspire/blob/main/src/Components/Telemetry.md?

Reply:
Done